Update strapi.mdx (#6762)
withastro/docs: src/content/docs/en/guides/cms/strapi.mdx

> Now, you can use this environment variable in your Astro project.

Reviewer: ```suggestion
Restart the dev server to use this environment variable in your Astro project.
```

If you would like to have IntelliSense for your environment variable, you can create a `env.d.ts` file in the `src/` directory and configure `ImportMetaEnv` like this:

> :::note
> .env files are loaded at the beginning of Vite's startup and changes take effect after restarting the server.
> :::

Reviewer: ```suggestion
```
